tf2_ros/src/buffer_server.cpp

```diff
@@ -232,4 +275,18 @@ geometry_msgs::msg::TransformStamped BufferServer::lookupTransform(GoalHandle gh
     goal->source_frame, tf2_ros::fromMsg(goal->source_time), goal->fixed_frame);
 }
 
+geometry_msgs::msg::TransformStamped BufferServer::lookupTransform(const std::shared_ptr<LookupTransformService::Request> request)
```

Comment:
I implemented this function similar to how it is for the action server, however I think we can simplify quite a bit if we use the lookupTransform from `buffer.cpp` instead of `buffer_core.cpp` -> `canTransform` is called in there so no need to call it explicitly + we could use the `timeout` argument instead of handling it again with `end time`. What do you think?

Reply:
That seems like a reasonable path to me.